actors/builtin/miner/miner_actor.go

```diff
@@ -357,9 +360,15 @@ func (a Actor) SubmitWindowedPoSt(rt Runtime, params *SubmitWindowedPoStParams)
 				params.Deadline, currEpoch, currDeadline.Index)
 		}
 
-		// Verify that the PoSt was committed to the chain at the challenge deadline
-		// (or at most WPoStChallengeLookback+WPoStChallengeWindow in the past).
-		commRand := rt.GetRandomnessFromTickets(crypto.DomainSeparationTag_PoStChainCommit, currDeadline.Challenge, nil)
+		// Verify that the PoSt was committed to the chain at most WPoStChallengeLookback+WPoStChallengeWindow in the past.
+		if params.ChainCommitEpoch < currDeadline.Challenge {
+			rt.Abortf(exitcode.ErrIllegalArgument, "expected chain commit epoch %d to be after %d", params.ChainCommitEpoch, currDeadline.Challenge)
+		}
+		if params.ChainCommitEpoch > currEpoch {
```

Comment:
Should be >=. Randomness in the current epoch isn't defined until the tipset is created (at which point it's no longer currrent)